pal/pacer/pacer-web.ts

```diff
@@ -106,25 +104,46 @@ export class Pacer {
         if (!this._isPlaying) return;
         this._ctTime(this._stHandle);
         this._stHandle = 0;
+        if (this._timeId) {
+            clearTimeout(this._timeId);
+            this._timeId = 0;
+        }
         this._isPlaying = false;
         this._frameCount = 0;
     }
 
     _handleRAF = (): void => {
-        const elapseTime = performance.now() - this._startTime;
+        if (!this._isPlaying) {
+            return;
+        }
+        const currTime = performance.now();
+        const elapseTime = currTime - this._startTime;
         const elapseFrame = Math.floor(elapseTime / this._frameTime);
         if (elapseFrame < this._frameCount) {
-            this._rAF.call(window, this._handleRAF);
-        } else {
-            this._frameCount++;
-            if (this._callback) {
-                this._callback();
+            const sleepMs = this._frameCount * this._frameTime - elapseTime;
+            if (sleepMs > 16) {
+                this._stHandle = this._rAF.call(window, this._handleRAF);
```

Comment:
If running on high frame rate devices, for example, 120FPS, the `requireAnimationFrame` interval is `8ms` instead of `16ms`. So  will `this._rAF.call` here be invoked twice ?

Reply:
I've tried searching for a way to get the screen refresh rate, but haven't found one yet. Therefore, at high resolution, use settimeout for delay